infra/tpu-pytorch/infra_triggers.tf

```diff
@@ -2,7 +2,7 @@ module "terraform_apply" {
   source = "../terraform_modules/apply_terraform_trigger"
 
   included_files    = ["infra/**"]
-  branch           = "master"
+  branch           = "main"
```

Comment:
This doesn't look right. Our primary branch is still `master`.

Reply:
Ah right! I merged main branch instead of master that updated this file. I'll revert it